packages/block-template-react/.bprc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "apiKey": "b10ck5.5551820619c3b59abcff6d07325cc6e6.e86fc0f4-2228-4d4d-96b5-6240bfee0745"
+}
```

Comment:
🚨

Reply:
Oops - thanks! I've been so careful to avoid `git add .` when jumping around but forgot it when sorting out merge conflicts on this one

The key is for my local db so fine to remain in history – have removed